cmd/taskctl/taskctl.go

```diff
@@ -225,7 +212,7 @@ func (tc *TaskCtlCmd) buildTaskRunner(args []string, conf *config.Config) (*runn
 	}
 
 	go func() {
-		<-cancel
+		<-cancel // tc.Cmd.Context().Done()
```

Comment:
Comment?

Reply:
I would want to keep these in here for now as they need to be properly implemented

listening to os.Signals isn't done properly and we don't shut down gracefully